amplify/backend/function/risestandstrongcb34053a/src/utils/aws-utils.js

```diff
@@ -100,6 +100,34 @@ async function getShift(startTimestamp) {
         let shift = await docClient.get(params).promise();
         return shift
     } catch (err) {
+        console.log(err);
+        throw err;
+    }
+}
+
+/**
+ * PUT Modifies a shift with the specified startTimestamp from the shifts table in DynamoDB.
+ * Any ommitted fields in the request body will not be updated.
+ * Returns nothing. Throws error from DynamoDB if one occurs.
+ * 
+ * @param {*} shift
+ */
+async function putShift(shift) {
```

Comment:
I think we should explicitly require startTimestamp here as a separate parameter because it's the sort key of our table and if they don't provide it to us this method will always error

Reply:
Thanks for reviewing -- could you help me with this part?